### PR TITLE
Add pip option

### PR DESCRIPTION
**Pull request by CagtayFabry**
adds the option to install some packages via `pip` instead of `conda`

### CODE REVIEW

**Comment by CagtayFabry** [comment]
The pip package gets installed but somehow micromamba doesn't list it in the environment

File: create_conda_env_file.py [hunk 1]
```diff
@@ -67,10 +68,18 @@ class MetadataType:
 # cleanup
 env["dependencies"] = [dep.replace(" ", "") for dep in env["dependencies"]]
 
+# pip installs
+pip = list(set(env["dependencies"]) & set(args.pip))
+env["dependencies"] = list(set(env["dependencies"]) - set(pip))
```
Comment (marscher) [inline create_conda_env_file.py:73]: ain't this missing the term `- set(args.pip)`? I suggest using variables for this.

Comment (marscher) [reply create_conda_env_file.py:73]: ah no, I'm wrong :)

**Comment by marscher** [comment]
> 

confirmed that micromamba does not list pip installed packages. I suggest adding a "pip list", if the pip option was active.

**Comment by CagtayFabry** [comment]
> > 
> 
> confirmed that micromamba does not list pip installed packages. I suggest adding a "pip list", if the pip option was active.

maybe we add that note in the README section of the `pip` option

**Review by marscher — APPROVED** [review]

**Pull request merged**